communications/newsletters/newsletter_12_Nov2019.md

```diff
@@ -0,0 +1,100 @@
+This DOI represents all versions, and will always resolve to the latest one.
+Different versions can be found on [Zenodo](https://zenodo.org) and can be individually cited.
+
+(Kirstie's very sorry that she's behind with releasing some of the versions 😩!
+If you attended a book sprint or have written chapters for the book your name will appear in that author list soon.
+Promise!)
+
+## Acknowledgements and celebrations
+
+Kirstie Whitaker presented a talk at the Open Science Community Nijmegen on 9 October 2019 under the title [*The Turing Way*: A global collaboration for reproducible and collaborative research](https://zenodo.org/record/3478387#.XcmkDZNKjUI), and at NESTA Hack STIR event on 22 October 2019 under the title [*The Turing Way*: A handbook for reproducible research](https://zenodo.org/record/3515937#.XcmkeJNKjUI).
+
+Thanks to several of you for joining Kirstie Whitaker, Yo Yehudi, and Malvika Sharan at the [MozFest House](https://ti.to/Mozilla/mozfesthouse-whyspace/en) to discuss and work on *The Turing Way* specific ideas, and discussing the project with us at the [MozFest 2019](https://www.mozillafestival.org/en/).
+
+![]()
+
+### Features and Shoutouts
+
+A nature article, [Make code accessible with these cloud services](https://www.nature.com/articles/d41586-019-03366-x), by Jeffrey Perkel acknowledged contributions of Turing Way team members, Sarah Gibson and Kirstie Whitaker’s contributions to the [BinderHub](https://binderhub.readthedocs.io/en/latest/) project.
+
+Oxford e-Research centre published an article [Transforming the culture of data science](https://www.oerc.ox.ac.uk/news/transforming-culture-data-science) based on the [impact story](https://www.turing.ac.uk/research/impact-stories/changing-culture-data-science), which highlighted the work and vision of this project.
+
+The project got shoutouts at [FORCE2019](https://www.force11.org/meetings/force2019) (see this [Tweet by Patricia Hertereich](https://twitter.com/PHerterich/status/1184034991400861696)), at [Goodenough College](https://www.goodenough.ac.uk/) in a [talk by Kirstie](https://zenodo.org/record/3522225#.XcmklpNKjUI), at [SciData19](https://researchdata.springernature.com/users/256983-irene-rodriguez/posts/50480-better-science-through-better-data-2019-scidata19-call-for-lightning-talks-proposal) by Yasemin Turkyilmaz-van der Velden in her talk [Reproducible Research: Why and How?](https://zenodo.org/record/3530485#.XcmopZNKjUI), by Tommy Hall at [Code Mesh London](https://codesync.global/conferences/code-mesh-ldn/) (see this [tweet by Evelina Gabasova](https://twitter.com/evelgab/status/1192804318795182080)) and at [EMBL](http://embl.org) Inspirational Seminar Series in Heidelberg ([talk by Kisrtie](https://zenodo.org/record/3532650#.XcmlAZNKjUI)).
+
+![]()
```

Comment:
What picture here?

Reply:
I hope you saw the Tweets (image in the google draft)

Reply:
Yep - so sorry about that